dask_planner/src/sql/logical/create_memory_table.rs

```diff
@@ -0,0 +1,78 @@
+use crate::sql::exceptions::py_type_err;
+use crate::sql::logical::PyLogicalPlan;
+use datafusion_expr::{logical_plan::CreateMemoryTable, logical_plan::CreateView, LogicalPlan};
+use pyo3::prelude::*;
+
+#[pyclass(name = "CreateMemoryTable", module = "dask_planner", subclass)]
+#[derive(Clone)]
+pub struct PyCreateMemoryTable {
+    create_memory_table: Option<CreateMemoryTable>,
+    create_view: Option<CreateView>,
+}
+
+#[pymethods]
+impl PyCreateMemoryTable {
+    #[pyo3(name = "getName")]
+    pub fn get_name(&self) -> PyResult<String> {
+        match &self.create_memory_table {
+            Some(create_memory_table) => Ok(format!("{}", create_memory_table.name)),
```

Comment:
`format!("{}",` is not needed, can just use `Ok(create_vire.name)`

Reply:
Cool, will open up a follow up to make this consistent across the Rust code as I took this pattern from another file